### ISSUE TITLE
Test coverage

### ISSUE THREAD

**Issue by ButuzGOL** [SpencerCDixon/react-testing-starter-kit#1]
Great starter!
The only thing is no coverage (

**Comment by SpencerCDixon** [comment]
Yeah sorry the name is probably somewhat misleading, this wasn't really meant to be a starter kit for production apps, just a repo to put the code associated with my blog post about getting testing set up in React.  If you're looking for a good starter kit that's production ready I really recommend this one: https://github.com/davezuko/react-redux-starter-kit
